arviz/plots/bf_plot.py

```diff
@@ -0,0 +1,48 @@
+# Plotting and reporting Bayes Factor given idata, var name, prior distribution and reference value
+def plot_bf(trace, var_name, prior, family = 'normal',  ref_val=0, xlim=None, ax=None):
+    # grab trace, a variable name to compute difference and prior.
+    # ref_val is the parameter we want to compare
+    from scipy import stats
+    import matplotlib.pyplot as plt
+    import numpy as np
+    # test some elemtns
+    # varName should be string
+    if not isinstance(var_name, str):
+        print('varName is not a string')
+    # BFs based on density estimation (using kernel smoothing instead of spline)
+    # stack trace posterior
+    tr = trace.posterior.stack(draws=("chain", "draw"))
+    post = tr[var_name]
```

Comment:
This could be 

```suggestion
    post = extract(idata, var_names=var_name)
    prior = extract(idata, var_names=var_name, group="prior")
```

Ref: https://python.arviz.org/en/latest/api/generated/arviz.extract.html.

We also generally use `data` or `idata` instead of `trace` and get the samples from the multiple groups from there.

Reply:
I've added these lines in the new version. But, as sometimes we would like to play with the prior, I've left the option to input a prior other than the one stated in the model